src/routes/routes.ts

```diff
@@ -0,0 +1,16 @@
+import { HomeController } from '../controllers/HomeController';
```

Comment:
Should we be consistent with the module naming here, for example, first letter being uppercase? Like the other modules within the project

Reply:
routes.ts -> Routes.ts

Reply:
Yes I agree, we should have consistent casing. Is it okay if we use CamelCasing?

Reply:
i don't see any problem with this :-)